test.js

```diff
@@ -10,6 +10,7 @@ const fixture = '🦄';
 
 test.beforeEach(t => {
 	t.context.conf = new Conf({cwd: tempy.directory()});
+	t.context.cryptoConf = new Conf({cwd: tempy.directory(), cryptoKey: 'abc123'});
```

Comment:
The instance should just be initiated in the test body, not here, since it's not shared anywhere else anymore.